### DISCUSSION TITLE
No Configured Instances

### DISCUSSION THREAD

**Discussion by ngervasio** [basetime/bldr-sfmc#20] [General]
I'm having trouble configuring an instance. I have created the installed app in sfmc and entered the required 4 fields that the cli prompts. I run 'bldr config -l' but the list of instance names is empty and I get the message 'No Configured Instances'.

**Comment by anthonymzupancic** [comment]
Sorry I just saw this!

Are you still having this issue? Release was pushed around this time so curious if it's still occurring.

I'm out of office right note but will look onto it asap!

**Reply by anthonymzupancic** [reply]
@ngervasio seems like there is an issue, I've added it as an issue here on the repo and will take a look into it Monday when I'm back in office.

**Reply by anthonymzupancic** [reply]
@ngervasio [here](https://github.com/basetime/bldr-sfmc/issues/21#issue-1266094967) is the issue I opened, can you provide me the Installed Package scope you are using?
I was able to get it working with a properly scoped package, but want to ensure I'm not ruling something out.

**Comment by anthonymzupancic** [comment]
@ngervasio updated the package and the issue [here](https://github.com/basetime/bldr-sfmc/issues/21#issue-1266094967).
